pkg/operator/configobservation/configobservercontroller/config_observer_controller.go

```diff
@@ -70,6 +72,7 @@ func NewConfigObserverController(
 		observeoauth.ObserveAccessTokenInactivityTimeout,
 		libgoetcd.ObserveStorageURLsToArguments,
 		encryptobserver.NewEncryptionConfigObserver("openshift-oauth-apiserver", "/var/run/secrets/encryption-config/encryption-config"),
+		featuregates.NewObserveFeatureFlagsFunc(nil, nil, []string{"apiServerArguments", "feature-gates"}, featureGateAccessor),
```

Comment:
Does it make sense to "allowlist" externalOIDC in particular, rather than having `allowAll`?

Reply:
I'm mostly interested in wiring the upstream CBOR apiserver and client-go gates to oauth-apiserver today. Ideally, each operator would know how to filter the features published in the OCP features API to only those its operand accepts, but I don't think we have a mechanism to do that except manually curating the list. I recently patched oauth-apiserver to filter out features it doesn't accept on its side as a temporary solution.

Reply:
This translates into, no thx? Because...

- We would need to manually curate that list (which sounds fine).
- The oauth-apiserver does this filtering and it is cooler, because...?

Reply:
> which sounds fine

I like your faith in us! I changed it to use a list.